controller/build.gradle

```diff
@@ -1,7 +1,8 @@
 apply plugin: 'java'
 apply plugin: 'maven'
-apply plugin: 'findbugs'
+//apply plugin: 'findbugs'
```

Comment:
'findbugs' is a static code analyser for detecting bugs. It's better to be included. I commented out when I was experimenting with json validation schemas because compiling with 'findbugs' takes a longer time. I'll add it back.